### PR TITLE
Do not use \tcode for non-syntax.

### LINKED ISSUE #106: Inconsistent formatting of "public" and "inline" in sentences
There are a bunch of inconsistencies around whether to `\tcode` `public` and `inline`, e.g. in "public inline member function" throughout the standards text.

### PR DESCRIPTION
**Pull request by jensmaurer**
This applies to inline, public, protected, private, and friend.

Fixes #106.

### CODE REVIEW

File: source/numerics.tex [hunk 1]
```diff
@@ -6752,12 +6752,6 @@
 matrix class nor a field class.
 However, it is a very useful building block for designing such classes.}
 
-\pnum
-An implementation is permitted to qualify any of the functions declared in
-\tcode{<valarray>}
-as
-\tcode{inline}.
-
```
Comment (Eelis) [inline source/numerics.tex]: Hmm, this is erasing the whole paragraph rather than just the tcode bit. Was that intended?

Comment (jensmaurer) [inline source/numerics.tex]: Yes, the statement is entirely redundant with [global.functions] p1

> It is unspecified whether any non-member functions in the C++ standard library are defined as inline (10.1.6).

and [member.functions] p1

> It is unspecified whether any member functions in the C++ standard library are defined as inline (10.1.6).

Comment (Eelis) [inline source/numerics.tex]: Ah ok, was just checking because this edit doesn't match the commit description.

**Comment by zygoloid** [comment]
Looks good, needs rebasing.

**Pull request merged**